apps/angular/src/core/dtos/anime.dto.ts

```diff
@@ -0,0 +1,54 @@
+// Because we get this type from backend
+/* eslint-disable @typescript-eslint/naming-convention */
+import { AnimeStatusDto } from './backend-enums/anime-status.dto';
+import { AnimeTypeDto } from './backend-enums/anime-type.dto';
+import { DateTimeRangeDto } from './date-time-range.dto';
+
+/** Represents anime in backend. */
+export type AnimeDto = {
+
+	/** Anime id. */
+	readonly id: number;
+
+	/**
+	 * Created.
+	 * @type {string} - String in ISO date format.
+	 */
+	readonly created: string;
+
+	/**
+	 * Modified.
+	 * @type {string} - String in ISO date format.
+	 * */
+	readonly modified: string;
+
+	/** Anime title in English. */
+	readonly title_eng: string;
+
+	/** Anime title in Japanese. */
+	readonly title_jpn: string;
+
+	/** Image - anime poster. */
+	readonly image: string;
```

Comment:
Let's add some clarification whether about the image.
I guess, it's a link to image.
It might be important because in some cases we might get image encoded in base64 or path on the server instead of a link (or something else).
```suggestion
	/** Anime poster image URL. */
	readonly image: string;
```